corehq/motech/forms.py

```diff
@@ -166,7 +166,7 @@ def helper(self):
                 ),
                 id="div_id_oauth_settings",
             ),
-            twbscrispy.PrependedText('skip_cert_verify', ''),
+            crispy.Field('skip_cert_verify'),
```

Comment:
Was this change to put the label to the right of the checkbox instead of in the column with the other labels? Does it look wrong if the label is in the label column?

Reply:
This was to put the checkbox in the right hand column where the other inputs are otherwise the checkbox is rendered in the left column (but it looks the same as in the screenshot, just in the left column)

Reply:
This usage is still incorrect? It should be `CheckboxField` from `hqcrispy`
https://github.com/dimagi/commcare-hq/blob/f9e382a88b273758f9050c956548e16b79314a40/corehq/apps/hqwebapp/crispy.py#L279-L280